tests/test.py

```diff
@@ -6173,6 +6165,41 @@ def testServerRead(self):
         super().testServerRead()
         self.assertIn("Finished processing client's connection", self.read_log(self.acra))
 
+class TestAcraIgnoresLegacyKeys(AcraCatchLogsMixin, BaseTestCase):
+    """
+    Ensure AcraServer won't exit with error when started with flags for key caching,
+    while keystore contains legacy keys (Connector<->Server, Connector<->Translator)
+    """
+
+    def checkSkip(self):
+        if KEYSTORE_VERSION != 'v1':
+            self.skipTest("test only for keystore v1")
+
+    def setUp(self):
+        super().setUp()
+
+        def create_key_file(name):
+            open(f"{KEYS_FOLDER.name}/{name}", "w").close()
```

Comment:
Do we really want to create some test oriented key files inside general `KEYS_FOLDER` directory. Probably it would be better to create some temporary directory and drop it when test is finished?